index.html

```diff
@@ -0,0 +1,186 @@
+              </ul>
+             <p class="content-work"> 
+                 A daily selection of privately personalized reads; no accounts or sign-ups required.  
+             </p>
+
+             <ul class="content-laguages">
+                <li class="lang"><a href="#">html</a></li>
+                <li class="lang"><a href="#">css</a></li>
+                <li class="lang"><a href="#">javascript</a></li>
+             </ul>
+             <div>
+                  <button type="submit" class="project-btn">See Project</button>
+             </div>
+        </div>
+        <div class="work-items">
+              <a class="design" href="#"> <img src="./images/art-printing.png" alt="Art printing"></a>
+                  
+              <h1 class="work-title">Multi-Post
+                                    <br>Stories </h1>
+               <ul class="projects">
+                   <li class="work-name"><a href="#">CANOPY</a></li>
+                   <li class="counter"><img src="./images/Counter.png"></li>
+                   <li><a class="backend-dev">Back End Dev</a></li>
+                   <li class="counter"><img src="./images/Counter.png"></li>
+                   <li><a class="backend-dev" href="#">2015</a></li>
+               </ul>
+               <p class="content-work"> 
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.  
+               </p>
+               <ul class="content-laguages">
+                 <li class="lang"><a href="#">html</a></li>
+                 <li class="lang"><a href="#">css</a></li>
+                 <li class="lang"><a href="#">javascript</a></li>
+               </ul>
+               <div>
+               <button type="submit" class="project-btn">See Project</button>
+               </div>
+         </div>
+     </section>
+     <section class="about-me">
+        <div class="about"> About <br>Myself
+            <p>I’m a software developer! I can help you <br> build a product, feature or website Look <br>
+              through some of my work and experience! If you like what you see and
+               have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+            <a href="#" class="let-connect">LET'S CONNECT </a>
+
+            <ul class="icons-social-1">
+                <li><a class="icon icon-Github" href="#"> <img src="../icon-social/Github.png" alt="Github"/><a></a></li>
+                <li><a class="icon icon-Linkedin" href="#"> <img src="../icon-social/Linkedin.png" alt="Linkedin"></a></li>
+                <li><a class="icon icon-iOS" href="#"> <img src="../icon-social/iOS.png" alt="iOS"></a> </li>
+                <li><a class="icon icon-Twitter" href="#"><img src="../icon-social/Twitter.png" alt="Twitter"></a></li>
+                <li><a class="icon icon-Messenger" href="#"> <img src="../icon-social/Messenger.png" alt="Messenger"/></a></li>
+           </ul>
```

Comment:
:red_circle:  As you can below none of these icons is displaying. You should keep one dot `.` before / to index the current directory. Kindly use one dot like this  `<img src=" ./icon-social/Github.png" alt="Github"/><a>` and not 
`<img src="../icon-social/Github.png" alt="Github"/><a>`. Please update all the images path under this  section :hugs: 

Your Design
![iconnotdispl](https://user-images.githubusercontent.com/26736582/200059478-3dfa82aa-4375-431f-93de-c2a332bb4145.png)